### PR TITLE
Fix many version fields before 1.0

### PR DESCRIPTION
**Pull request by Dazpoet**
I faked a 1.0 install to see what would show up in CKAN. This is a first batch of fixed version fields in .ckan files handpackaged for CKAN-meta or in some cases older files generated from NetKAN.

Since some of them tell me who to blame I'll throw some blame on @Felger and @hakan42 for doing mistakes anyone could have done! Or maybe I'm the one doing mistakes by marking mods not working for 1.0 when in reality they'll be fine?

Basically I'm changing mods to stop their compatibility at 0.90 instead of being compatibile for "any" version or just anything from version X through the usage of `ksp_version_min`.

This is related to https://github.com/KSP-CKAN/NetKAN/issues/1002

TL;DR: I'm adding a metric tonne of `"ksp_version_max" : "0.90",` lines

### CODE REVIEW

**Comment by Dazpoet** [comment]
Failed due to astronomers pack being... well astronomers pack and overwriting and mucking about with files all over the place. Seeing as all .ckan files validated fine and litterally nothing has been changed except the version fields I'd say this is ready for a merge.

**Comment by Dazpoet** [comment]
So 1.0 is coming tomorrow, this is just a start and more is needed before metadata is secured but needs to be done incrementally so I can track progress. I'm merging this and if it fails horribly I'll revert it.
